Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.py

```diff
@@ -586,29 +587,35 @@ def get_ticket_attachments(self, ticket_id: str) -> dict:
         """
         return self.send_request('attachment', 'GET', params={'sysparm_query': f'table_sys_id={ticket_id}'})
 
-    def get_ticket_attachment_entries(self, ticket_id: str) -> list:
+    def get_ticket_attachment_entries(self, ticket_id: str, get_timestamp: bool = False) -> list:
```

Comment:
as discussed, lets query attachments by timestamp instead of returning the `sys_created_on` for all and iterate them

Reply:
Which timestamp?

Reply:
`sys_created_on`